backend/src/database_pg.ts

```diff
@@ -103,15 +121,42 @@ export class Database {
         }
     }
 
-    public async close() {
-        try {
-            await this.pool.end();
-            console.log('Closed the database connection.');
-        } catch (err: any) {
-            console.error('Error closing the database connection:', err.message);
+    public async deletePackagebyID(packageID: string){
+        const sql = `DELETE FROM packages_table WHERE id=$1`
+        try{
+            const res = await this.pool.query(sql, [packageID]);
+            Logger.logInfo(`Deleted ${packageID} from packages_table database`)
+            
+        } catch(err: any){
+            console.error(`Error deleting ${packageID}`, err.message); 
+            throw err; 
         }
     }
 
+    public async deletePackagebyName(packageName: string){
+
+        const sql = `SELECT id from packages_table WHERE name=$1`;
+
+        try{
+            const res = await this.pool.query(sql, [packageName]);
+
+            if(!res){return null;}
+            
+            res.rows.forEach(async(row) => {
+
+                await this.deletePackagebyID(row.id);
+
+            })
+            
```

Comment:
can you add a logging statement that shows how many rows were deleted? this would be useful for once we go into version testing